config/software/td-agent-files.rb

```diff
@@ -9,7 +9,7 @@
 build do
   block do
     # setup related files
-    pkg_type = project.packagers_for_system.first.id.to_s
+    pkg_type = project.packager.id.to_s
```

Comment:
This is no longer relevant. Per discussion, we are gonna drop support of Ruby version < 2.2 for this packaging repo. So I reverted this change.